www/docs/en/dev/guide/platforms/android/index.md

```diff
@@ -127,15 +129,17 @@ On a Mac or Linux, with a text editor, create or modify the `~/.bash_profile` fi
 To set an environment variable, add a line that uses `export` like so (substitute the path with your local installation):
 
 ```bash
-export ANDROID_SDK_ROOT=/Development/android-sdk/
+export ANDROID_HOME=/Development/android-sdk/
 ```
 
 To update your `PATH`, add a line resembling the following (substitute the paths with your local Android SDK installation's location):
 
 ```bash
-export PATH=$PATH:$ANDROID_SDK_ROOT/platform-tools/
-export PATH=$PATH:$ANDROID_SDK_ROOT/cmdline-tools/latest/bin/
-export PATH=$PATH:$ANDROID_SDK_ROOT/emulator/
+export PATH=$PATH:$ANDROID_HOME/platform-tools/
+export PATH=$PATH:$ANDROID_HOME/cmdline-tools/latest/bin/
+export PATH=$PATH:$ANDROID_HOME/tools/bin/
```

Comment:
The `tools/bin` path is actually obsolete (these are the binaries for Java 8), so this line can be removed.

cmdline-tools package is the replacement which is already listed above.